components/Randomizer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react'
+import { Image } from 'react-native'
+
+import VillagerArray from './VillagerArray'
+import { StartState } from './StartState'
+import { DetectShake } from './DetectShake'
+import { Container, Name, Quote, TextContainer, Textspan } from './ProfileSyles'
+
+export const Randomizer = () => {
+  const [villager, setVillager] = useState([])
+  let villagerIcon = villager.uri
+
+ // Function to fetch random villager from array
+  const getVillager = () => {
+    const theVillager = VillagerArray[Math.floor(Math.random() * VillagerArray.length)]
+    setVillager(theVillager)
+  }
+
+  // Start = No villager is selected: Instruction text and placeholder image displayed.
+  const [start, setStart] = useState(true)
+  const shake = () => {
+    setStart(false)
+  }
+  useEffect(() => {
+    DetectShake.addListener(() => {
+      shake()
+      getVillager()
+      console.log('shaken not stirred')
+    })
+      return () => {
+        DetectShake.removeListener()
+      }
+  }, [])
+  return (
+    <>
+      {start ? (
+        <StartState />
+      ) : (
+        <Container>
+        <Image 
+          source={{uri: villagerIcon}}
+          style={{
+            width: 120,
+            height: 120,
```

Comment:
Those styles should be moved to styled component

Reply:
I tried to move this image styling to ProfileStyles.js but it broke something. :(